community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -152,6 +150,13 @@ def _find_dynamic_node_status() -> NodeAction:
     # * delete orhpaned instances
     return NodeActionUnchanged()  # don't touch dynamic nodes
 
+def get_fr_action(nodename:str, state:NodeState) -> Optional[NodeAction]:
+    fr = lookup().future_reservation(lookup().node_nodeset(nodename))
+    now = datetime.utcnow()
```

Comment:
If reservation is dormant and node is DOWN return NodeActionUnchanged

Reply:
Added, now its if reservation is dormant and node is not DOWN make it go DOWN otherwise return NodeActionUnchanged